### PR TITLE
Update saphostctrl parser and sap combiner

### PR DESCRIPTION
**Pull request by xiangce**
- https://bugzilla.redhat.com/show_bug.cgi?id=1547563
- Both the combiner and parser are newly developed
- rename the python file and class name
- remove the inheritance between Lssap and the SAPHostCtrlInstance

### CODE REVIEW

**Comment by eduardocerqueira** [comment]
test this please

**Comment by eduardocerqueira** [comment]
The CI failed above is not related to change on this PR. It was a change in the insights-client that changed the status return and we are updating our tests now.

File: insights/parsers/saphostctrl.py [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+        'D90'
+        >>> sap_inst.data[-1]['SapVersionInfo']  # Note: captured as one string
+        '749, patch 211, changelist 1754007'
+        >>> sap_inst.data[0]['InstanceType']  # Inferred code from InstanceName
+        'HDB'
+
+    Attributes:
+        instances (list): The list of instances found in the cluster output.
+        sids (list): The list of SID found in the cluster output.
+        types (list): The list of instance types found in the cluster output.
```
Comment (bfahr) [inline insights/parsers/saphostctrl.py]: @xiangce you should also define the `data` attribute since that is what you show in the examples.

Comment (xiangce) [reply insights/parsers/saphostctrl.py]: @bfahr , Added as per your suggestion, Thanks.

**Comment by xiangce** [comment]
test this please

**Comment by esakaiev** [comment]
test this please

**Pull request merged**